extendr-api/src/functions.rs

```diff
@@ -4,7 +4,7 @@ use crate::*;
 ///
 /// Use the Env{} wrapper for more detail.
 /// ```
-/// use extendr_api::*;
+/// use extendr_api::prelude::*;;
```

Comment:
This file contains some double `;`. Maybe you mistyped a regex replacement?

Reply:
Very likely!